core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -41,21 +41,13 @@
 #include <vector>
 
 /*--------------------------------------------------------------------------*/
-namespace Kokkos {
-namespace Impl {
-
-inline bool execute_in_serial(OpenMP const& space = OpenMP()) {
-  return (OpenMP::in_parallel(space) &&
-          !(omp_get_nested() && (omp_get_level() == 1)));
-}
-
-}  // namespace Impl
-}  // namespace Kokkos
 
 namespace Kokkos {
 namespace Impl {
 
-class OpenMPInternal;
+class OpenMPInternal {
+  return m_pool[i];
+};
```

Comment:
What happened here?
```suggestion
class OpenMPInternal;
```

Reply:
I think some of our changes were lost in the rebasing.